.lintstagedrc

```diff
@@ -4,5 +4,5 @@
     "prettier --write",
     "jest --findRelatedTests"
   ],
-  "*.md": ["prettier --write"]
+  "*.{md,yml}": ["prettier --write"]
```

Comment:
We need to make this formatting and the ones within `package.json` `scripts` consistent since the latter is only formatting inside docs, lib, and tests folders.